src/basic/pidfd-util.c

```diff
@@ -245,3 +243,32 @@ int pidfd_get_inode_id(int fd, uint64_t *ret) {
                 *ret = (uint64_t) st.st_ino;
         return 0;
 }
+
+int pidfd_get_inode_id_self_cached(uint64_t *ret) {
```

Comment:
This fits in pidfd-util better still in my eyes

Reply:
hmm? it *is* in pidfd-util?

Reply:
I guess I really need to restore decent sleep and stop submitting reviews on the phone ..